nss-day-cohort-60/national-parks-api
# added data to Wildlife_Groups table

Data added to the Wildlife_Groups table.
Steps to test:
1. Check for spelling and other errors in the data in national_park.sql
2. Make sure data is displaying in the table in national_park.sqlite3
